src/index.ts

```diff
@@ -20,7 +21,7 @@ import { ComponentsHelper } from './utils/components-helper.service';
 @NgModule({
   exports: [
     AccordionModule, AlertModule, ButtonsModule, CarouselModule, CollapseModule, DatepickerModule, DropdownModule,
-    ModalModule, PaginationModule, ProgressbarModule, RatingModule, TabsModule, TimepickerModule, TooltipModule,
+    ModalModule, PaginationModule, ProgressbarModule, RatingModule, SortableListModule, TabsModule, TimepickerModule, TooltipModule,
```

Comment:
I am planning to include a lot of non bootstrap code modules here
but we should not blow up core module